src/lexer.coffee

```diff
@@ -782,15 +803,12 @@ RESERVED = [
   'protected', 'public', 'static'
 ]
 
-STRICT_PROSCRIBED = ['arguments', 'eval', 'yield*']
+STRICT_PROSCRIBED = ['arguments', 'eval']
```

Comment:
(It’s in `JS_KEYWORDS` (“keywords that CoffeeScript shares in common with JavaScript”).)